docs/classes/ApplicationMetadata.html

```diff
@@ -55,7 +55,7 @@
 					<a href="../modules.html">amazon-chime-sdk-js</a>
 				</li>
 				<li>
-					<a href="applicationmetadata.html">ApplicationMetadata</a>
+					<a href="ApplicationMetadata.html">ApplicationMetadata</a>
```

Comment:
Why do those doc files change? Are those expected?

Reply:
I don't know but think so, the `npm run build:release` on updated them which is the directions in the PR